deploy/packer/base.json

```diff
@@ -44,6 +44,12 @@
     {
       "type":           "shell",
       "inline":         ["sudo unattended-upgrade"]
+    },
+    {
+      "type": "shell",
```

Comment:
Already discussed on IRC, but commenting here for posterity. The commit already references the cve and the bug, so a comment is redundant.